tensorflow/core/kernels/mkl_matmul_ops_common.h

```diff
@@ -707,8 +707,8 @@ void dnnl_gemm_batch(const std::vector<bool>& transa,
                      const std::vector<int>& n, const std::vector<int>& k,
                      const std::vector<float>& alpha, const T* a, const T* b,
                      const std::vector<float>& beta, T* c,
-                     const int group_count,
-                     const std::vector<int>& group_size) {
+                     const int group_count, const std::vector<int>& group_size,
+                     OpKernelContext* ctx = nullptr) {
```

Comment:
Why does `dnnl_gemm_batch` take `OpKernelContext` if it's not going to use it?

Reply:
I see now that the rest of the changes to `dnnl_gemm_batch` is in https://github.com/tensorflow/tensorflow/pull/40254. Is it possible to clean this PR up so they don't overlap? If it's inconvenient, then I'm okay with leaving it like this.

Between this and https://github.com/tensorflow/tensorflow/pull/40254, which one should be merged first? Or any order is fine?

Reply:
unfortunately they shared a common file. There is no ordering requirement. Both PR's are independent as they contain the required changes. Merging in any order should be fine. There should not be any conflicts since the changes are exactly same.
Thanks.